typescript/sdk/src/deploy/HyperlaneProxyFactoryDeployer.ts

```diff
@@ -48,14 +48,14 @@ export class HyperlaneProxyFactoryDeployer extends HyperlaneDeployer<
       this.factories,
     ) as (keyof ProxyFactoryFactories)[]) {
       const factory = await this.deployContract(chain, factoryName, []);
-      this.addVerificationArtifacts(chain, [
-        {
-          name: proxyFactoryImplementations[factoryName],
-          address: await factory.implementation(),
-          constructorArguments: '',
-          isProxy: true,
-        },
-      ]);
+      const artifact = {
+        name: proxyFactoryImplementations[factoryName],
+        address: await factory.implementation(),
+        constructorArguments: '',
+        isProxy: true,
+      };
+      await this.verifyContract(chain, artifact);
+      this.addVerificationArtifacts(chain, [artifact]);
```

Comment:
how come we still need to add the artifacts?

Reply:
this is my trying to comply with good practice of minimizing backward incompatible changes